### ISSUE TITLE
[CAAnimation] has no member moveBy

### ISSUE THREAD

**Issue by quanguyen** [IBAnimatable/IBAnimatable#522]
Hi,

I just installed v5.0.0 from v4.x and got this issue:

<img width="917" alt="screen shot 2017-11-07 at 3 36 13 pm" src="https://user-images.githubusercontent.com/11938078/32484178-7b6a3390-c3d1-11e7-9d05-9ce3653d10cb.png">

Where: 
`fileprivate let controller = AnimatableView()`

How should I fix this bug? (This code was working well in v4.x)

Thanks,

**Comment by JakeLin** [comment]
@quanguyen `.moveBy` is `AnimationType` not `CAAnimation`. Can you have a look at `AnimationsTimingFunctionViewController`. It has an example using it, thanks.

```
private func starFall(_ view: AnimatableView) {
    let moveY = view.superview?.frame.height ?? 140
    view.animate(.moveBy(x: 0, y: Double(moveY - 40))).completion {
      if #available(iOS 10.0, *) {
        Timer.scheduledTimer(withTimeInterval: 1, repeats: false) { _ in
          view.transform = .identity
        }
      } else {
        view.transform = .identity
      }
    }
  }
```

**Comment by quanguyen** [comment]
Thanks for the tip, Jake,

I notice that my previous code does not work due to `.then`. It should be modified to: 

```
controller.animate(.moveBy(x: Double(frame.width - controller.frame.width), y: 0))
            .completion { [weak self] in
            guard let strongSelf = self else { return }
            
            strongSelf.controller.animate(.moveBy(x: 0, y: 0))
                .completion({
                strongSelf.activity.startAnimating()
            })
        }
```

Is it true `.then` syntax is no more supported?

**Comment by JakeLin** [comment]
@quanguyen it is still supported. Can you have a look at the playground file `Chaining Animations`

```
view.animate(.squeezeFade(way: .in, direction: .down))
    .then(.pop(repeatCount: 1))
    .then(.shake(repeatCount: 1))
    .then(.squeeze(way: .in, direction: .down))
    .then(.wobble(repeatCount: 1))
    .then(.flip(along: .x))
    .then(.flip(along: .y))
    .then(.slideFade(way: .out, direction: .down))
```

You can also change `AnimationsTimingFunctionViewController` like

```
view.animate(.moveBy(x: 0, y: Double(moveY - 40)))
      .then(.moveBy(x: 0, y: 0))
      .completion {
      if #available(iOS 10.0, *) {
        Timer.scheduledTimer(withTimeInterval: 1, repeats: false) { _ in
          view.transform = .identity
        }
      } else {
        view.transform = .identity
      }
      }
```
I can't figure out why your one is not working 😞

**Comment by tbaranes** [comment]
@quanguyen Closing this due to the inactivity, don't hesitate to reopen it if you have another question!
